kasa/protocol.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         self._port = config.port_override or self.default_port
         self._credentials = config.credentials
         self._credentials_hash = config.credentials_hash
-        self._timeout = config.timeout
+        self._timeout = config.timeout or self.DEFAULT_TIMEOUT
```

Comment:
Isn't this already handled by the deviceconfig class?

Reply:
Yes but this stops all sorts of annoying mypy issues with it thinking timeout is optional